README.md

```diff
@@ -8,142 +16,181 @@
 
 Documentation is available at http://docs.elemeno.io
 
-## API Overview
+## Usage
 
 ```js
-var elemeno = require('elemeno');
+var Elemeno = require('elemeno');
+
+var elemeno = new Elemeno('123e4567-e89b-12d3-a456-426655440000');
 
-// Set API Key
-elemeno.setAPIKey('123e4567-e89b-12d3-a456-426655440000');
+elemeno.getCollectionItems('recipes', function(err, response) {
+	if (err) return console.log(err);
+
+	// Do something with the response
+	console.log(response);
+});
 ```
 
+## Caching
+
+You have the option of caching API requests locally in memory by passing in cache options when creating your `elemeno` object. 
+
+**Note:** The respone from the Elemeno API is also cached on the Elemeno's servers for 2 minutes independent of the cache settings used on this client library. Please take this into consideration when configuring your local cache.
```

Comment:
This reads a little weird: 
"... cached on the Elemeno's servers"

Maybe "cached on Elemeno's servers"... or "cached on the Elemeno server"

I'm sure you intended one of those, but your brain jammed 'em together.